client/src/modules/users/users.js

```diff
@@ -64,6 +64,14 @@ function UsersController($state, $uibModal, Users, Notify, Modal, uiGridConstant
         enableFiltering : true,
         cellClass : muteDisabledCells,
       },
+      {
+        field : 'lastLogin',
+        type : 'date',
+        displayName : 'USERS.LAST_LOGIN',
+        cellFilter : 'date:"dd/MM/yyyy HH:mm:ss"',
```

Comment:
I like including the time.  I do wonder whether the 24 hour time should be used or the 12 hour time +am/pm?  I think Congo/French-speaking users are used to 24 hour time, but perhaps in the future this should be a regional option?